static/admin/config.yml

```diff
@@ -12,10 +12,11 @@ public_folder: /img
 collections:
   - name: 'story'
     label: 'Story'
+    extension: 'json'
     create: true
-    folder: 'static/stories.json'
+    folder: 'static/stories'
     editor:
-      preview: true
+      preview: false
```

Comment:
Why we gotta do a false preview?

Reply:
I wanted to see how that changes anything.